src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,60 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (newMovie: Movie) => void;
+};
+
+const clearMovieForm = {
+  title: '',
+  description: '',
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [movieForm, setMovieForm] = useState(clearMovieForm);
+
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  } = movieForm;
+
+  const submitDisabled = !title.trim() || !imgUrl.trim()
```

Comment:
```suggestion
  const isSubmitDisabled = !title.trim() || !imgUrl.trim()
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/344352c0-2cc7-4aff-92b3-82e5c2ec76b5)